runtime/src/lib.rs

```diff
@@ -28,6 +28,7 @@ use frame_support::{
 	},
 	PalletId,
 };
+use frame_support::traits::ConstU32;
```

Comment:
There's alredy another `frame_support::traits` import above. Try to put new import there.